prov/efa/src/efa_shm.c

```diff
@@ -87,15 +87,13 @@ int efa_shm_ep_name_construct(char *smr_name, size_t *smr_name_len, struct efa_e
 	return FI_SUCCESS;
 }
 
-struct fi_info *g_shm_info;
-
 /**
- * @brief initliaze the global variable g_shm_info
- *
- * g_shm_info is used to create shm resources (fabric,domain,endpoint,cq)
- *
+ * @brief Create a shm info object based on application info
+ * 
```

Comment:
Extra whitespace

Reply:
Fixed.

Reply:
It looks like there's still whitespace there

Reply:
Oops, should be fixed now.